web/assets/js/views/PhysicalEvaluation-0.3-ehr.js

```diff
@@ -728,8 +728,12 @@ PMI.views['PhysicalEvaluation-0.3-ehr'] = Backbone.View.extend({
         this.finalized = obj.finalized;
         this.rendered = false;
         this.render();
+        var today = new Date();
+        var sixMonthsAgo = (new Date()).setMonth(today.getMonth() - 6);
```

Comment:
Took me a while to figure out how this was working. TIL [negative month offsets in JavaScript](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/setMonth#parameters).

Reply:
Yep. Can't recall where I saw that trick, but it helps with writing a bit cleaner code.